examples/pylab_examples/quiver_demo.py

```diff
@@ -8,76 +8,89 @@
 The workaround is to manually expand the axes.
 
 '''
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
 from numpy import ma
 
-X, Y = meshgrid(arange(0, 2*pi, .2), arange(0, 2*pi, .2))
-U = cos(X)
-V = sin(Y)
+X, Y = np.meshgrid(np.arange(0, 2 * np.pi, .2), np.arange(0, 2 * np.pi, .2))
+U = np.cos(X)
+V = np.sin(Y)
 
-#1
-figure()
-Q = quiver(U, V)
-qk = quiverkey(Q, 0.5, 0.92, 2, r'$2 \frac{m}{s}$', labelpos='W',
-               fontproperties={'weight': 'bold'})
-l, r, b, t = axis()
+# 1
+plt.figure()
+Q = plt.quiver(U, V)
+qk = plt.quiverkey(Q, 0.5, 0.92, 2, r'$2 \frac{m}{s}$', labelpos='W',
+                   fontproperties={'weight': 'bold'})
+l, r, b, t = plt.axis()
 dx, dy = r - l, t - b
-axis([l - 0.05*dx, r + 0.05*dx, b - 0.05*dy, t + 0.05*dy])
+plt.axis([l - 0.05 * dx, r + 0.05 * dx, b - 0.05 * dy, t + 0.05 * dy])
```

Comment:
It looks like this step is creating a 5% margin. Unfortunately, it does not look like you would get the same effect if you do `plt.margins(0.05)` instead of the previous 3 lines involving `axis()`... I wonder if that is a bug?